plugins/README.md

```
@@ -143,3 +143,51 @@ NOTE: large db may GC overhead limit exceeded.
+   - View the generated keystore file name and wallet address
+3. Enter `ImportPrivateKey` to import an existing private key, you will be prompted to:
+   - Input the 64-character hex private key
+   - Input password (hidden input)
+   - Input password again for confirmation
+   - View the generated keystore file name and wallet address
+4. Enter `Exit` or `Quit` to exit the tool.
+
+### Output:
+
+Keystore files are saved in the `Wallet` directory with the following naming convention:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)